### ISSUE TITLE
[Accessibility] [Password page] [AxeDev Tools] Language button has no discernible text

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#866]
**Describe the bug**
The language button has no id or label. The screen reader doesn't identify what is the component uses and the user can't change the available resource (in this case, change language).

Ensure buttons have discernible textmore informationLink opens in a new windowElement Location:.z-20To solve this problem, you need to fix at least (1) of the following:Element does not have inner text that is visible to screen readersaria-label attribute does not exist or is emptyaria-labelledby attribute does not exist, references elements that do not exist or references elements that are emptyElement has no title attributeElement's default semantics were not overridden with role="none" or role="presentation"

**To Reproduce**
1. Go to 'https://muda.cambiatus.io/join?redirect=%2Fdashboard'
2. Click on "I'm already a member" button
3. Run AxeDev Tools
4. Check the component without label or id

**Expected behavior**
Form components must have id and label to work for screen readers, as 2.5.3 WCAG criteria.

**Screenshots**
![image](https://github.com/user-attachments/assets/e1a5fdc3-6aef-421b-992d-a526066ffbc8)

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**

- OS: Windows
- Browser Chrome
- Version 129.0.6668.101 (Versão oficial) 64 bits

**Comment by ceciliabelem** [comment]
For mapping all defects statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
